app/mailers/rent_mailer.rb

```diff
@@ -4,11 +4,11 @@ class RentMailer < ApplicationMailer
   #
   #   en.rent_mailer.send.subject
   #
-  def new_rent(email, first_name, title, from, to)
-    @from = from
+  def new_rent(email, first_name, title, to, locale)
+    I18n.locale = locale
```

Comment:
Consider replacing the setting of the locale by assigning it like this. Doing `I18n.locale = locale` will set the locale for the whole application, and leak into subsequent requests to the API, until you set the locale again.

Think of this scenario:

A user with locale `:en` rents a book, the locale is set accordingly and an email is sent saying:
```
Your book has been rented!
```

Two seconds later, a user with locale `:es` wants to retrieve a book that the API can't find, and gets this error message:

```
Error: book not found
```

The message should return in Spanish, since his locale is set to `:es`, but the locale hasn't been changed since the last time someone rented a book.

To avoid this, try using the `I18n.with_locale` method, with something like this:

```ruby
I18n.with_locale(locale) do
  @to = to
  @title = title
  @first_name = first_name
  mail to: email, subject: t(:new_rent)
end
```

Reply:
Done, thanks